test/unit/option_spec.lua

```diff
@@ -0,0 +1,61 @@
+local helpers = require("test.unit.helpers")
+
+local to_cstr = helpers.to_cstr
+local eq      = helpers.eq
+
+local option = helpers.cimport("./src/nvim/option.h")
+local globals = helpers.cimport("./src/nvim/globals.h")
+
+local check_ff_value = function(ff)
+  return option.check_ff_value(to_cstr(ff))
+end
+
+local get_sts_value = function()
+  return option.get_sts_value()
+end
+
+describe('check_ff_value', function()
+
+  it('views empty string as valid', function()
+    eq(1, check_ff_value(""))
+  end)
+
+  it('views "unix" as valid', function()
+    eq(1, check_ff_value("unix"))
+  end)
+
+  it('views "dos" as valid', function()
+    eq(1, check_ff_value("dos"))
+  end)
+
+  it('views "mac" as valid', function()
+    eq(1, check_ff_value("mac"))
+  end)
+
+  it('views "foo" as invalid', function()
+    eq(0, check_ff_value("foo"))
+  end)
+end)
+
+describe('get_sts_value', function()
+  it([[returns 'softtabstop' when it is non-negative]], function()
+    globals.curbuf.b_p_sts = 5
+    eq(5, get_sts_value())
+    globals.curbuf.b_p_sts = 0
+    eq(0, get_sts_value())
+  end)
+
+  it([[returns "effective shiftwidth" when 'softtabstop' is negative]], function()
+    local shiftwidth = 2
+    globals.curbuf.b_p_sw = shiftwidth
+    local tabstop = 5
+    globals.curbuf.b_p_ts = tabstop
+
+    globals.curbuf.b_p_sts = -2
+    eq(shiftwidth, get_sts_value())
+
+    shiftwidth = 0
+    globals.curbuf.b_p_sw = shiftwidth
+    eq(tabstop, get_sts_value())
+  end)
```

Comment:
I think  changing to `expected_shiftwidth` and `expected_tabstop` would add very little here. The reader already expects the first parameter of `eq` to be the expected value and second the actual.